src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -74,6 +80,19 @@ public boolean includeNs() {
     }
 
     /**
+     * Does this tool emit information for uncovered loci? Tools that do should override to return {@code true}.
+     *
+     * NOTE:  Typically, this should only be used when intervals are specified.
+     * NOTE:  If MappedReadFilter is removed, then emitting empty loci will fail.
+     *
+     * @return {@code true} if this tool requires uncovered loci information to be emitted, {@code false} otherwise
+     */
+    public boolean emitEmptyLoci() {
```

Comment:
Please see issues #2678 and #2679 

We will address this in a separate pull request that will possibly refactor the code changes in this PR.